packages/admin-ui/src/lib/core/src/extension/components/route.component.ts

```diff
@@ -53,5 +64,14 @@ export class RouteComponent {
         this.title$ = combineLatest([inject(ROUTE_COMPONENT_OPTIONS).title$, breadcrumbLabel$]).pipe(
             map(([title, breadcrumbLabel]) => title ?? breadcrumbLabel),
         );
+
+        this.locationId = this.route.snapshot.data.locationId;
```

Comment:
Is there a default `locationId` or can this be `undefined`?

Reply:
both `locationId` and `description` can be `undefined`